### PR TITLE
[elasticsearch] Add timeout configuration support

### PR DESCRIPTION
**Pull request by consulthys**
## Proposed commit message

The [`timeout` option](https://www.elastic.co/guide/en/beats/metricbeat/current/configuration-metricbeat.html#_timeout) is available when configuring the Metricbeat `elasticsearch` module. This PR adds the possibility of configuring the `timeout` option for the corresponding `elasticsearch` integration.

## Checklist

- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

Closes https://github.com/elastic/integrations/issues/11467

### CODE REVIEW

File: packages/elasticsearch/manifest.yml [hunk 1]
```diff
@@ -78,6 +78,14 @@ policy_templates:
             required: true
             show_user: false
             default: node
+          - name: timeout
+            type: text
```
Comment (sachin-frayne) [inline packages/elasticsearch/manifest.yml:82]: why is this of type `text`?

**Review by sachin-frayne — APPROVED** [review]
Other than my understanding of why you need `text` for a timeout, this LGTM!

File: packages/elasticsearch/manifest.yml (hunk 1 above)
Comment (consulthys) [reply packages/elasticsearch/manifest.yml:82]: Because the values are like "10s"

**Pull request merged**